metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/configs/HiveConnectorFastServiceConfig.java

```diff
@@ -36,6 +40,7 @@
  * @since 1.1.0
  */
 @Configuration
+@EnableTransactionManagement(proxyTargetClass = true)
```

Comment:
Do you want to enable this on a conditional class? This configuration class won't be loaded unless that `useHiveFastService` is enabled. I assume you currently only want it for your jdbc transactions you're using but if you ever try to use transactions without this flag enabled they're coupled together.

Reply:
At present, useHiveFastService only will be using the database connections directly and so will use the transaction management.